src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup/RecoveryServicesBackupCmdletBase.cs

```diff
@@ -121,9 +121,9 @@ private void HandleException(Exception exception)
             }
         }
 
-        public override void ExecuteCmdlet()
+        protected override void BeginProcessing()
         {
```

Comment:
You need to also change in all cmdlets to call BeginProcessing instead of ExecuteCmdlet